app/src/main/java/org/pesmypetcare/mypetcare/services/UserManagerAdapter.java

```diff
@@ -90,6 +90,7 @@ public boolean changePassword(User user, String newPassword) {
     public void deleteUser(User user) {
         try {
             ServiceLocator.getInstance().getUserManagerClient().deleteUser(user.getToken(), user.getUsername());
+            //ServiceLocator.getInstance().getUserManagerClient().deleteUserFromDatabase(user.getToken(), user.getUsername());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 126).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5355440)